src/main/webapp/i18n/de/legal.json

```diff
@@ -1,10 +1,26 @@
 {
-    "legal": {
-        "imprint": {
-            "title": "Impressum"
-        },
-        "privacy": {
-            "title": "Datenschutz"
+    "artemisApp": {
+        "legal": {
+            "language": {
+                "de": "Deutsch",
+                "en": "Englisch"
+            },
```

Comment:
We probably have these translation strings elsewhere already I guess? Maybe just use those?

Reply:
I intentionally introduced them because they are specific to the legal documents. 
In theory, we could offer a document in a language that is not supported by the entire platform.